src/main/webapp/app/exam/participate/exercises/text/text-exam-submission.component.ts

```diff
@@ -35,7 +37,7 @@ export class TextExamSubmissionComponent extends ExamSubmissionComponent impleme
     private textEditorInput = new Subject<string>();
 
     // Icons
-    farListAlt = faListAlt;
+    readonly farListAlt = faListAlt;
```

Comment:
not a major thing and should not block the PR from being merged, but if you need to push again  you could make this one protected aswell